classes/controller/front.ctrl.php

```diff
@@ -410,6 +397,57 @@ public function render_form($item, $errors)
         return \View::forge($layout['layout'], $layout['args'], false);
     }
 
+    /**
+     * @param Model_Field $field : The select field
+     * @param $value
+     * @return array : An associative array like : [$value-1 => $label-1, ...]
+     */
+    protected function getSelectFieldChoices(Model_Field $field, &$value)
+    {
+        if ($field->field_technical_id === 'recipient-list') {
```

Comment:
To improve lisibility, can you define a variable : 

```
$isRecipientList = ($field->field_technical_id === 'recipient-list');
```

because you do this test 3 times.